srcpkgs/please/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'please'
+pkgname=please
+version=0.3.16
+revision=1
+_commit="d330c5b24018ad6144eeb12f382ca5c45c885640"
+wrksrc="${pkgname}-${_commit}-${_commit}"
+build_style=cargo
+makedepends="cmake llvm clang pam-devel"
+depends="pam"
```

Comment:
The `pam-libs` dependency is detected automatically, does it also rely on `pam` itself?